demisto_sdk/commands/content_graph/strict_objects/integration.py

```diff
@@ -0,0 +1,160 @@
+from pathlib import Path
+from typing import Optional, Any
+from pydantic import Field
+from demisto_sdk.commands.common.constants import MarketplaceVersions
+from demisto_sdk.commands.common.tools import get_file
+from demisto_sdk.commands.content_graph.strict_objects.base_strict_model import BaseStrictModel, CommonFields, \
+    Argument, Output, Important, ScriptType, BaseIntegrationScript
+
+
+class Configuration(BaseStrictModel):
+    display: Optional[str] = None
+    section: Optional[str] = None
+    advanced: Optional[str] = None
+    default_value: Optional[Any] = Field(None, alias="defaultvalue")
+    name: str
+    type: int
+    required: Optional[bool] = None
+    hidden: Optional[Any] = None
+    hidden_x2: bool = None
```

Comment:
todo to the both of us, check whether `_x2` fields are in use

Reply:
Who should we check it with?

Reply:
1. try removing them and see how validate all behaves only with this validation
2. DM

Reply:
hidden_x2 was removed from both Configuration and Command schemas